docs/src/api/params.md

```diff
@@ -639,14 +639,14 @@ If no origin is specified, the username and password are sent to any servers upo
 * langs: js, java
 - `colorScheme` <null|[ColorScheme]<"light"|"dark"|"no-preference">>
 
-Emulates `'prefers-colors-scheme'` media feature, supported values are `'light'`, `'dark'`, `'no-preference'`. See
+Emulates `'prefers-colors-scheme'` media feature, supported values are `'light'` and `'dark'`. See
```

Comment:
to keep it short, I didn't include the "no-preference is deprecated" info in this one. Interested folks can figure that out by following the "see Page.emulateMedia" link.